### PR TITLE
login-page

### PR DESCRIPTION
**Pull request by Shailseen**
image

### CODE REVIEW

**Review by aurobindoGupta — APPROVED** [review]
nice work!! please add screenshot

**Comment by Shailseen** [comment]
>

**Review by Aditi0505 — APPROVED** [review]
LGTM

**Review by prakashsakari — APPROVED** [review]
Looks good to me

File: pages/homepage/login-page.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+}
+.card-container{
+    width: 22rem;
+}
+.card-container h2{
+    text-align: center;
+    margin-bottom: 0rem;
+}
+.login-container{
+    display: flex;
```
Comment (arpit01923) [inline pages/homepage/login-page.css:22]: put this in flex in a common class

File: pages/homepage/login-page.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    flex-direction: column;
+    padding: 1rem 3rem 3rem 3rem;
+}
+.form-input{
+    display: flex;
```
Comment (arpit01923) [inline pages/homepage/login-page.css:27]: same here

File: pages/homepage/login-page.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    font-weight: 600;
+    padding: 0rem;
+    margin-bottom: 2px;
+}
+.login-container .solid-button{
+    background-color: var(--login-primary-color);
+    width: 100%;
+}
+.form-utils{
+    display: flex;
```
Comment (arpit01923) [inline pages/homepage/login-page.css:49]: same

File: pages/homepage/login-page.css [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    color: var(--black-color);
+}
+.create-accnt-container > i{
+    color: var(--black-color);
+    font-size: 1.5rem;
+    margin-top: 1px;
+
+}
+.create-accnt-container{
+    display: flex;
```
Comment (arpit01923) [inline pages/homepage/login-page.css:78]: same

File: pages/homepage/login-page.html [hunk 5]
```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (arpit01923) [inline pages/homepage/login-page.html:6]: add author and description of meta tags

**Review by arpit01923 — COMMENTED** [review]
some minor changes

**Pull request merged**